src/JSONFormatter.cpp

```diff
@@ -0,0 +1,35 @@
+#include "JSONFormatter.hpp"
+#include "Log.hpp"
+
+namespace signal_estimator {
+
+void JSONFormatter::report_latency(double sw_hw, double hw, int sma_window, double hw_avg) {
+    if (first_output) {
+        se_log_info("[");
+        first_output = false;
+    }
+
+    se_log_info("  {\"sw+hw\": %7.3f, \"hw\": %7.3f, \"hw_avg%d\": %7.3f},", sw_hw,
+        hw, sma_window, hw_avg);
```

Comment:
I think we should switch from se_log_info to printf in both formatters. se_log_info prints to stderr, but ideally we should print only logs to stderr and the measurement results should be printed to stdout, so that they can be easily redirected to file.